build/common/installer/scripts/tomlparser.rb

```diff
@@ -21,6 +21,7 @@
 @collectClusterEnvVariables = true
 @logTailPath = "/var/log/containers/*.log"
 @logExclusionRegexPattern = "(^((?!stdout|stderr).)*$)"
+@logStitchMultiline = false
```

Comment:
Hi, @gracewehner , I am planning to add the automation script to deploy all the workloads and scenarios yamls to the scale cluster so that we can use them to scale test and also potentially for any dev/test cluster to validate, can you please add the test yamls related to multiline logging so that we can use them to test on regular basis?